### ISSUE TITLE
Customtkinter v5 crashes TkinterMapview

### ISSUE THREAD

**Issue by felle9900** [TomSchimansky/CustomTkinter#716]
Today I updated my version of Customtkinter to v5, but now my TkinterMapView crashes my app.
It worked fine yesterday before I updated.

I get this error:
```
File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/customtkinter/windows/widgets/core_widget_classes/ctk_base_class.py", line 153, in cget
    raise ValueError(f"'{attribute_name}' is not a supported argument. Look at the documentation for supported arguments.")
ValueError: 'bg' is not a supported argument. Look at the documentation for supported arguments.
```

**Comment by Akascape** [comment]
This is all because version 5 replaced old arguments like bg, text_font etc with new names.

**Comment by TomSchimansky** [comment]
I will fix tkintermapview soon!

**Comment by TomSchimansky** [comment]
It's fixed now with tkintermapview version 1.18.
